app/src/main/java/com/peacecorps/pcsa/circle_of_trust/CircleOfTrustFragment.java

```diff
@@ -0,0 +1,120 @@
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_circle_of_trust, container, false);
+        requestButton = (ImageButton) rootView.findViewById(R.id.requestButton);
+        editButton = (ImageButton) rootView.findViewById(R.id.editButton);
+        editButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                startActivity(new Intent(getActivity(),Trustees.class));
+            }
+        });
+        requestButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showOptions();
+            }
+        });
+        return rootView;
+    }
+
+    public void showOptions(){
+        AlertDialog.Builder builderSingle = new AlertDialog.Builder(
+                getActivity());
+        builderSingle.setTitle("Select a request");
+        final ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.select_dialog_singlechoice);
+        arrayAdapter.add("Come get me");
+        arrayAdapter.add("Call I need an interruption");
+        arrayAdapter.add("I need to talk");
+        builderSingle.setNegativeButton("cancel",
+                new DialogInterface.OnClickListener() {
+
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        dialog.dismiss();
+                    }
+                });
+
+
+        builderSingle.setAdapter(arrayAdapter,
+                new DialogInterface.OnClickListener() {
+
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        optionSelected = arrayAdapter.getItem(which);
+                        sendMessage(optionSelected);
+                    }
+                });
+        builderSingle.show();
+    }
+
+    public void sendMessage(String optionSelected)
+    {
+        SmsManager sms = SmsManager.getDefault();
+        String message = "";
+        switch(optionSelected)
+        {
+            case "Come get me":
+                message = "Come and get me. I need help getting home Safely. Call ASAP to get my Location.Sent through First Aide's Circle of Trust.";
+                break;
+            case "Call I need an interruption":
+                message = "Call and pretend you need me. I need an interruption. Message sent through First Aide's Circle of Trust";
+                break;
+            case "I need to talk":
+                message = "I need to talk. Message sent through First Aide's Circle of Trust";
+                break;
+        }
+
+        try {
+            // The numbers variable holds the Comrades numbers
+            String numbers[] = {"+2348103425729", "+2348111875720"};
```

Comment:
If you want to change the number you have to go through the code and change it. To avoid that use the string.xml file. Do this whenever you encounter a hard coded string. It is a simple best practice in general coding and specially in Android.